### PR TITLE
Spelling errors

### PR DESCRIPTION
**Pull request by brianteeman**
Redo of #11295

### CODE REVIEW

**Comment by jeckodevelopment** [comment]
I have tested this item :white_check_mark: successfully on 43d92c54b0841a547a95a2e84ab35f3cbdb9287a<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/11604">issues.joomla.org/joomla-cms/11604</a>.</sub>

File: administrator/components/com_finder/models/index.php [hunk 1]
```diff
@@ -240,9 +240,9 @@ protected function getListQuery()
 	}
 
 	/**
-	 * Method to get the state of the Smart Search plug-ins.
+	 * Method to get the state of the Smart Search Plugins.
 	 *
-	 * @return  array  Array of relevant plug-ins and whether they are enabled or not.
+	 * @return  array  Array of relevant Plugins and whether they are enabled or not.
```
Comment (andrepereiradasilva) [inline administrator/components/com_finder/models/index.php]: why the uppercase here?

**Comment by brianteeman** [comment]
4 people managed to review the original pr this is just a redo to avoid merge conflicts

File: libraries/joomla/linkedin/groups.php [hunk 2]
```diff
@@ -215,10 +215,10 @@ public function getSettings($person_id = null, $group_id = null, $fields = null,
 	 *
 	 * @param   string   $group_id          The unique identifier for a group.
 	 * @param   boolean  $show_logo         Show group logo in profile.
-	 * @param   string   $digest_frequency  E-mail digest frequency.
-	 * @param   boolean  $announcements     E-mail announcements from managers.
+	 * @param   string   $digest_frequency  email digest frequency.
+	 * @param   boolean  $announcements     email announcements from managers.
 	 * @param   boolean  $allow_messages    Allow messages from members.
-	 * @param   boolean  $new_post          E-mail for every new post.
+	 * @param   boolean  $new_post          email for every new post.
```
Comment (andrepereiradasilva) [inline libraries/joomla/linkedin/groups.php]: When starting the line shouldn't be uppercase?

File: libraries/joomla/linkedin/groups.php [hunk 3]
```diff
@@ -283,10 +283,10 @@ public function changeSettings($group_id, $show_logo = null, $digest_frequency =
...
```
Comment (andrepereiradasilva) [inline libraries/joomla/linkedin/groups.php]: When starting the line shouldn't be uppercase?

**Comment by infograf768** [comment]
Have anyone really tested all? This is touching a lot of files.

**Comment by andrepereiradasilva** [comment]
I have tested this item :white_check_mark: successfully on 32cbd3adaabcaded4ef6a3db7ee9e90671032d4b<br /><br />On code review. Just comments and string changes.<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/11604">issues.joomla.org/joomla-cms/11604</a>.</sub>

**Comment by brianteeman** [comment]
Thanks for testing @andrepereiradasilva @jeckodevelopment

**Comment by brianteeman** [comment]
@rdeutz should be ready to merge now

**Comment by brianteeman** [comment]
Thanks

**Pull request merged**